meetup/services/meetup_api_response_parser.py

```diff
@@ -0,0 +1,11 @@
+def parse(api_response):
+  json = api_response.json()
+  parsed_response = {}
+
+  if json:
+    parsed_response['venue'] = json[0].get('venue').get('name')
```

Comment:
Lots of duplication on every one of these lines. Could extracting a helper method help reduce it?